extensions/fairifier-rdf-extension/dtls/fairifier/RdfSkeletonTransformer.java

```diff
@@ -4,11 +4,38 @@
 import java.io.IOException;
 import java.nio.file.Path;
 
+/**
+ * Allows a developer to implement transformations from the openrefine provided
+ * JSON blob and other formats.
+ * 
+ * @author Shamanou van Leeuwen
+ * @since 5 Oktober 2017
+ * @see http://openrefine.org/
+ */
 public interface RdfSkeletonTransformer {
 
+    /**
+     * 
+     * Should return a Model object based on the Path given.
+     * 
+     * @param path
+     * @return Model
+     * @throws -
+     *             thrown when there is file available
```

Comment:
No exception specified, and it's not a proper sentence.

Reply:
Edits are in javadoc before this part